### PR TITLE
kubeedge source code analysis blog

### PR DESCRIPTION
**Pull request by WormOn**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [*] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (docs )

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: content/zh/blog/source-code-analysis/kubeedge-architecture.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+
+## 组件的共用框架和功能
+
+在源码层面，kubeedge核心独立组件包括cloudcore、edgecore、edge_mesh和edge_site，除此之外还有mappers和keadm，具体如下下表：
+
+|组件名|组件功能|备注|
+|:---:|:---:|:---:|
+|cloudcore|Cloud部分各功能模块的集合||
+|edgecore|Cloud部分各功能模块的集合||
+|edge_mesh|服务网格解决方案|源码目录中缺少makefile文件|
```
Comment (fisherxu) [inline content/zh/blog/source-code-analysis/kubeedge-architecture.md]: s/edge_mesh/edgemesh

File: content/zh/blog/source-code-analysis/kubeedge-architecture.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+|edge_site|边缘独立集群解决方案||
```
Comment (fisherxu) [inline content/zh/blog/source-code-analysis/kubeedge-architecture.md]: s/edge_site/edgesite

File: content/zh/blog/source-code-analysis/kubeedge-architecture.md [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+2. Edgesite 为满足在边缘需要完整集群功能的场景，定制的在边缘搭建既能管理、编排又能运行负责的完整集群解决方案；
+
+## 组件的共用框架和功能
+
+在源码层面，kubeedge核心独立组件包括cloudcore、edgecore、edge_mesh和edge_site，除此之外还有mappers和keadm，具体如下下表：
+
+|组件名|组件功能|备注|
+|:---:|:---:|:---:|
+|cloudcore|Cloud部分各功能模块的集合||
+|edgecore|Cloud部分各功能模块的集合||
```
Comment (fisherxu) [inline content/zh/blog/source-code-analysis/kubeedge-architecture.md]: Edge部分

File: content/zh/blog/source-code-analysis/kubeedge-architecture.md [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+|mappers|物联网协议实现包|本源码分析系列不涉及|
+|keadm|kubeedge的一键部署工具|目前支持unbuntu，本源码分析系列不涉及|
+
+以上组件中的cloudcore、edgecore、edge_mesh和edge_site具有类似的代码结构，具体如下表：
+
+|组件名|代码目录|组件启动入口|
+|:---:|:---:|:---:|
+|cloudcore|kubeedge/cloud|kubeedge/cloud/cloudcore/cloudcore.go，kubeedge/cloud/admission/admission.go，kubeedge/cloud/csidriver/csidriver.go|
+|edgecore|kubeedge/edge|kubeedge/edge/cmd/edgecore/edgecore.go|
+|edge_mesh|kubeedge/edgemesh|kubeedge/edgemesh/cmd/edgemesh.go|
```
Comment (fisherxu) [inline content/zh/blog/source-code-analysis/kubeedge-architecture.md]: edgemesh

File: content/zh/blog/source-code-analysis/kubeedge-architecture.md [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+|edge_site|kubeedge/edgesite|kubeedge/edgesite/cmd/edgesite.go|
```
Comment (fisherxu) [inline content/zh/blog/source-code-analysis/kubeedge-architecture.md]: edgesite

**Comment by WormOn** [comment]
The first article of kubeedge source code analysis!

**Comment by fisherxu** [comment]
@WormOn Overall looks good, could you please squash the commits?

**Comment by WormOn** [comment]
> @WormOn Overall looks good, could you please squash the commits?

OK.

**Pull request closed without merging**